jit/analyze.ts

```diff
@@ -76,14 +79,44 @@ module J2ME {
     "gnu/testlet/vm/NativeTest.returnAfterPause.()I": YieldReason.Root,
     "gnu/testlet/vm/NativeTest.dumbPipe.()Z": YieldReason.Root,
     "gnu/testlet/TestHarness.getNumDifferingPixels.(Ljava/lang/String;)I": YieldReason.Root,
+
+
   };
 
-  export function isFinal(classInfo: ClassInfo): boolean {
-    if (classInfo.isFinal) {
-      return true;
+  export var yieldVirtualMap = {
+    // These can technically yield but are worth the risk.
+    // "java/lang/Object.equals.(Ljava/lang/Object;)Z": YieldReason.None
```

Comment:
Or perhaps not worth the risk? Would be good to have a comment here explaining why this is commented out.